wiki/osu!tourney/en.md

```diff
@@ -7,285 +7,22 @@ Users encountering **problems** with the client may read the Troubleshooting sec
 ## What is osu!tourney?
 
 The osu!tourney client is the official streaming client used to spectate all players in a multiplayer room simultaneously.
-It is, currently, only meant to be used for world cups and might therefore be limited in its behaviour.
+It is, currently, only meant to be used for world cups and might therefore be limited in its behaviour. (You can still use it personally though, read multiplayer usage for more info.)
```

Comment:
remove the side note... the sentence is saying that the osu!tourney client is intended for osu! world cups, not personal use. This means that if something doesn't work (or breaks), it was because it wasn't meant to do that.